src/main/kotlin/com/ekino/oss/errors/generator/ErrorBodyGenerator.kt

```diff
@@ -102,6 +104,7 @@ private fun toError(
     errors = errors,
     globalErrors = globalErrors,
     service = service,
-    stacktrace = stacktrace
+    stacktrace = stacktrace,
+    timestamp = Instant.now(Clock.systemDefaultZone())
```

Comment:
Hi,

Thanks for the PR. I think we should stick to UTC here.

Reply:
Ok, thanks for your return.